### DISCUSSION TITLE
Can I set a custom project domain? / can a custom domain just be the project name without other additions like name-project.globeapp.dev

### DISCUSSION THREAD

**Discussion by ClarissaGazalaEvanthe** [invertase/globe#21] [Q&A] [answered]
![Screenshot from 2023-12-19 11-41-13](https://github.com/invertase/globe/assets/84509715/b04138b7-5f6a-4461-8c72-afb98b8cf04a)

I tried to create a project, but when I opened the domain: https://slebwe.globeapp.dev/ not found

domain only: https://slebwe-yhjyij9-gwcangtip.globeapp.dev/

**Comment by Ehesp** [comment, marked as answer]
Hey, 

This is something we need to make clearer. There's two questions here:

1. Custom domains are on the roadmap, will be implemented soon.
2. Each deployment gets a unique url for the project. If you push to production (`--prod` flag on CLI), the domain you see there will be used for that deployment. Without the `--prod` flag, deployments are classed as preview and do not get the main domain assigned.

**Reply by ClarissaGazalaEvanthe** [reply]
Thank you, now I can use the project domain name
